CMakeLists.txt

```diff
@@ -499,33 +499,33 @@ endif()
 # Generate source files #
 #########################
 
-if (UA_ENABLE_FULL_NS0)
+set(UA_FILE_NODEIDS ${PROJECT_SOURCE_DIR}/deps/ua-nodeset/Schema/NodeIds.csv)
+set(UA_FILE_STATUSCODES ${PROJECT_SOURCE_DIR}/deps/ua-nodeset/Schema/StatusCode.csv)
+set(UA_FILE_TYPES_BSD ${PROJECT_SOURCE_DIR}/deps/ua-nodeset/Schema/Opc.Ua.Types.bsd)
```

Comment:
Note that we had a discussion that it should be possible to build the minimal nodeset without checking out the ua-nodeset submodule.
See also:
https://github.com/open62541/open62541/commit/c205c4d5455a66a27dab32f9a303db4908139475

I'm fine if we always depend on the submodule :+1:

Reply:
Reversed.